applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -319,6 +304,62 @@ config BT_AUDIO_BROADCAST_24_2_2
 	  24kHz, 48kbps, 4 retransmits, 60ms transport latency, and 40ms presentation delay
 endchoice
 
+config BT_AUDIO_BROADCAST_NAME
+	string "Broadcast name"
+	depends on TRANSPORT_BIS
+	default "NRF5340_BROADCASTER"
+	help
+	  Name of the broadcast, not the same as BT_DEVICE_NAME
```

Comment:
```suggestion
	  Name of the broadcast; not the same as BT_DEVICE_NAME.
```

Reply:
Sorry, for some reason I thought we didn't have punctuation at the end in the Kconfig, only the .rst, must have mixed it with the Doxygen doc. Will fix